tests/test_ops.py

```diff
@@ -17,31 +17,30 @@
 class TestOPSExpression(object):
 
     @pytest.mark.parametrize('equation, expected', [
-        ('Eq(u,3*a - 4**a)', 'Eq(ut0[OPS_ACC0(0)], -2.97015324253729)'),
+        ('Eq(u,3*a - 4**a)', 'Eq(ut0(0), -2.97015324253729)'),
         ('Eq(u, u.dxl)',
-         'Eq(ut0[OPS_ACC0(0)], -2.0*ut0[OPS_ACC0(-1)]/h_x + '
-            '0.5*ut0[OPS_ACC0(-2)]/h_x + 1.5*ut0[OPS_ACC0(0)]/h_x)'),
-        ('Eq(v,1)', 'Eq(vt0[OPS_ACC0(0,0)], 1)'),
+         'Eq(ut0(0), 0.5*ut0(-2)/h_x - 2.0*ut0(-1)/h_x + 1.5*ut0(0)/h_x)'),
+        ('Eq(v,1)', 'Eq(vt0(0, 0), 1)'),
         ('Eq(v,v.dxl + v.dxr - v.dyr - v.dyl)',
-         'Eq(vt0[OPS_ACC0(0,0)], 2.0*vt0[OPS_ACC0(0,-1)]/h_y - '
-            '0.5*vt0[OPS_ACC0(0,-2)]/h_y - 2.0*vt0[OPS_ACC0(0,1)]/h_y + '
-            '0.5*vt0[OPS_ACC0(0,2)]/h_y - 2.0*vt0[OPS_ACC0(-1,0)]/h_x + '
-            '0.5*vt0[OPS_ACC0(-2,0)]/h_x + 2.0*vt0[OPS_ACC0(1,0)]/h_x - '
-            '0.5*vt0[OPS_ACC0(2,0)]/h_x)'),
+            'Eq(vt0(0, 0), -0.5*vt0(0, -2)/h_y '
+            '+ 2.0*vt0(0, -1)/h_y - 2.0*vt0(0, 1)/h_y '
+            '+ 0.5*vt0(0, 2)/h_y + 0.5*vt0(-2, 0)/h_x '
+            '- 2.0*vt0(-1, 0)/h_x + 2.0*vt0(1, 0)/h_x '
+            '- 0.5*vt0(2, 0)/h_x)'),
         ('Eq(v,v**2 - 3*v)',
-         'Eq(vt0[OPS_ACC0(0,0)], vt0[OPS_ACC0(0,0)]**2 - 3*vt0[OPS_ACC0(0,0)])'),
+         'Eq(vt0(0, 0), vt0(0, 0)**2 - 3*vt0(0, 0))'),
         ('Eq(v,a*v + b)',
-         'Eq(vt0[OPS_ACC0(0,0)], 1.43*vt0[OPS_ACC0(0,0)] + 9.87e-7)'),
+         'Eq(vt0(0, 0), 1.43*vt0(0, 0) + 9.87e-7)'),
         ('Eq(v.dt,1/v + v**-1)',
-         'Eq(-vt0[OPS_ACC1(0,0)]/dt + vt1[OPS_ACC0(0,0)]/dt, 2/vt0[OPS_ACC1(0,0)])'),
+         'Eq(-vt0(0, 0)/dt + vt1(0, 0)/dt, 2/vt0(0, 0))'),
         ('Eq(w,c*w**2)',
-         'Eq(wt0[OPS_ACC0(0,0,0)], 999999999999999*wt0[OPS_ACC0(0,0,0)]**2)'),
+         'Eq(wt0(0, 0, 0), 999999999999999*wt0(0, 0, 0)**2)'),
         ('Eq(w,u + v )',
-         'Eq(wt0[OPS_ACC0(0,0,0)], ut0[OPS_ACC1(0)] + vt0[OPS_ACC2(0,0)])'),
-        ('Eq(u.forward,u+1)', 'Eq(ut1[OPS_ACC0(0)], ut0[OPS_ACC1(0)] + 1)'),
+         'Eq(wt0(0, 0, 0), ut0(0) + vt0(0, 0))'),
+        ('Eq(u.forward,u+1)', 'Eq(ut1(0), ut0(0) + 1)'),
         ('Eq(u, u.laplace)',
-         'Eq(ut0[OPS_ACC0(0)], ut0[OPS_ACC0(-1)]/h_x**2 - '
-            '2.0*ut0[OPS_ACC0(0)]/h_x**2 + ut0[OPS_ACC0(1)]/h_x**2)'),
+         'Eq(ut0(0), ut0(-1)/h_x**2 - '
+            '2.0*ut0(0)/h_x**2 + ut0(1)/h_x**2)'),
     ])
     def test_ast_convertion(self, equation, expected):
```

Comment:
maybe it will fork here if we will cover both old and new OPS syntax
@FabioLuporini should we cover both styles or aim to drop the old one? 
i'm asking this because i believe it could drastically impact our current efforts

Reply:
@vmickus fyi

Reply:
I'm almost done with the next PR (if I'm quick enough I might be able to add it on top of this) were the expressions are put in a callable and placed in the header and the parameters extracted. This should make using this very easy

Reply:
I'm happy to drop the old one, as long as this doesn't cause excessive pain to you guys @btrb .